charts/pega/templates/_pega-eks-ingress.tpl

```diff
@@ -52,9 +52,15 @@ spec:
   # To access the below service, along with {{ .node.domain }}, alb http port also has to be provided in the URL.
   - host: {{ template "domainName" dict "node" .node }}
     http:
-      paths:
-      - backend:
-          serviceName: {{ .name }}
+      paths: 
+      {{ if and .root.Values.constellation (eq .root.Values.constellation.enabled true) (eq .node.nodeType "WebUser") }}
+      - path: /prweb/constellation     
```

Comment:
Nigel says this probably needs to be a parameter - the same value we pass to the Docker image?